DEVELOPING_TUTORIALS.md

```diff
@@ -602,6 +612,73 @@ The properties exported from this file depend on the lesson type, as follows:
   - `question`: String - mandatory - _the question to be answered by the user_ ([detailed docs](#create-multiple-choice-quizzes-in-your-javascript-file-skip-for-coding-exercises-and-text-only-lessons))
   - `choices`: Array - mandatory - _the potential answers between which the user must choose_ ([detailed docs](#create-multiple-choice-quizzes-in-your-javascript-file-skip-for-coding-exercises-and-text-only-lessons))
 
+### `utils` module
+
+This module is a set of utils designed to be re-used accross the validation code of different tutorials.
+
+- `utils.format`
+
+  - `ipfsObject(object)`: formats an ipfs object to be ready to print in the output ui, for example, formatting a `cid` object into a string.
+
+  ```js
+  return {
+    success: 'Success! You did it!',
+    logDesc: "Here is the result.",
+    log: utils.format.ipfsObject(result)
+  }
+  ```
+
+- `utils.ipfs`
+
+  - `errorCodes`: set of error codes from `js-ipfs`
+    - `ERR_MORE_THAN_ONE_ROOT`
+
+  ```js
+  if (utils.ipfs.errorCodes.ERR_MORE_THAN_ONE_ROOT) {
+    return {
+      fail: 'Some error message specific to this edge case.'
+    }
+  }
+  ```
+
+  
+
+- `utils.validationMessages`
+
```

Comment:
Add context that these produce messages shown to the user - author still has to built the code that does the testing

Reply:
show an example of messages shown to user